### DISCUSSION TITLE
Can't log in to copilot from VScode in MacOS ventura 13.1

### DISCUSSION THREAD

**Discussion by JhonatanHern** [community/community#44529] [Copilot Conversations]
### Select Topic Area

Bug

### Body

When trying to log in to github copilot after opning VScode I'm getting the next error:

`Writing login information to the keychain failed with error 'Password is required.'.`

as shown in here:
![image](https://user-images.githubusercontent.com/25697703/212778758-5e2259d2-5aa6-4b9b-9df3-6c0015df3cdc.png)

I already paid for copilot and this error does not show up in the troubleshooting section. What could I do to solve it?

Thanks in advance.

**Comment by iamcjwilliams** [comment]
@JhonatanHern Have you tried uninstalling and reinstalling the extension?

**Reply by rdvo** [reply]
does this fix it? im having issues with this & my github pull/push repo extension.. what do i do ?

**Reply by iamcjwilliams** [reply]
1. Try signing in and out of your GitHub account and it will prompt you to sign back
2. If you are still experiencing issues provide a screenshot of your log or steps to replicate the error you're receiving

**Comment by rdvo** [comment]
this worked. thanks for your help


On Mon, Jan 23, 2023 at 9:23 AM iamcjwilliams ***@***.***>
wrote:

>
>    1. Try signing in and out of your GitHub account and it will prompt
>    you to sign back
>    2. If you are still experiencing issues provide a screenshot of your
>    log or steps to replicate the error you're receiving
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/44529#discussioncomment-4757004>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEZFGYHKGVWND4PNHTZNNDLWT2H5ZANCNFSM6AAAAAAT5HEIT4>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
*Robert DeVivo*
Founder

*T: 347.879.0056*
*E: ***@***.*** ***@***.***>*
*http://www.mtbmedia.com <http://www.mtbmedia.com>*

**Comment by carltheperson** [comment]
Uninstalling and reinstalling didn't work for me. But following the instructions in [this issue comment](https://github.com/orgs/community/discussions/18228#discussioncomment-3135528) did.

**Comment by bogdanAG22** [comment]
I'm logging in but I have errors, and it doesn't let me log in again
